src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationImprintViewModel.swift

```diff
@@ -0,0 +1,72 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import Foundation
+import UIKit
+
+class AppInformationImprintViewModel {
+	
+	static let englishContactFormLink = "https://www.rki.de/SharedDocs/Kontaktformulare/en/Kontaktformulare/weitere/Corona-Warn-App/Corona-Warn-App_Integrator.html"
+	static let germanContactFormLink = "https://www.rki.de/SharedDocs/Kontaktformulare/weitere/Corona-Warn-App/Corona-Warn-App_Integrator.html"
+	static var preferredLocalization = Bundle.main.preferredLocalizations.first ?? "de"
```

Comment:
I see what you did here.

I would prefer it that you inject this value into the ViewModel during the `init()` something along the lines of `init(preferredLocalization: String = "de")`

Imagine you would like to have two instances of `AppInformationImprintViewModel` with two different locals. Currently this would not be possible.

Reply:
done.